extension/src/components/Room.tsx

```diff
@@ -49,6 +49,25 @@ export default function Room({
     duration: number | undefined | null;
 }) {
     const isLoadingGlobal = useIsMutating();
+    const [autoScroll, setAutoScroll] = useState(true);
+    const [showNotification, setShowNotification] = useState(false);
```

Comment:
do we need to use 2 state variables? Why can't we also just use `autoScroll` for determining when to show the notification?